experimental/python/pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[project]
+name = "databricks-bundles"
+description = "Python support for Databricks Asset Bundles"
+version = "0.243.0-dev"
+
+authors = [
+    { name = "Gleb Kanterov", email = "gleb.kanterov@databricks.com" },
+]
+
+readme = "README.md"
+requires-python = ">=3.10"
+license = {file = "LICENSE"}
+
+dependencies = [
+]
+
+[tool.uv]
+dev-dependencies = [
+  "pyright==1.1.380",
+  "pytest-cov==5.0.0",
+  "pytest==8.3.3",
+  "ruff==0.6.8",
+  "typing_extensions==4.12.2",
+]
+
+[tool.setuptools]
+py-modules = ["databricks"]
+
+[tool.setuptools.package-data]
+"databricks.bundles" = ["py.typed"]
+
+[tool.pytest.ini_options]
+addopts = "-ra -q -vv"
+testpaths = [
+    "databricks_tests",
+]
+
+[tool.pyright]
+exclude = [
+    "**/__pycache__",
+    ".venv",
+]
+
+reportMissingImports = true
+reportMissingTypeStubs = false
+
+pythonVersion = "3.10"
+pythonPlatform = "Linux"
```

Comment:
Why not run it on Mac / Windows as well? This could detect reliance on Linux-specific APIs.

Reply:
I will set to "All", it will be the right setting to check against common denominator